### ISSUE TITLE
Add OMNY

### ISSUE THREAD

**Issue by jackmerrill** [Politiwatch/privacyspy#142]
**What's the product you'd like PrivacySpy to add?**
I want OMNY added to PrivacySpy because I have heard suspicious data concerns and skimmed the privacy policy.

**Where is the product's privacy policy? (Give a URL.)**
https://omny.info/privacy-policy/420

**Have you considered adding this product yourself?**
I can't read privacy policies deeply.

**Comment by milesmcc** [comment]
Yep, if someone would like to contribute the policy, we would gladly review it.
